src/Persistent_cohomology/include/gudhi/Persistent_cohomology.h

```diff
@@ -582,9 +582,11 @@ class Persistent_cohomology {
    * @return A vector of Betti numbers.
    */
   std::vector<int> betti_numbers() const {
-    // Init Betti numbers vector with zeros until Simplicial complex dimension and don't allocate a vector of negative
-    // size for an empty complex
-    std::vector<int> betti_numbers(std::max(dim_max_, 0));
+    // Return an empty vector for an empty complex
```

Comment:
The reference to an "empty *complex*" seems imprecise here.

Reply:
You are right. I rephrase the comment and add a test for Betti numbers when persistence is not computed